src/Particular.CodeRules/AnalyzerReleases.Shipped.md

```diff
@@ -4,20 +4,44 @@
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|--------------------
-PCR0001 |  Code    |  Warning | Await used without specifying ConfigureAwait
+PCR0001 |  Code    |  Warning | UseConfigureAwait
 
 ## Release 0.2
 
 ### New Rules
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|--------------------
-PCR0002 |  Code    |  Error   | Await or Capture Tasks
+PCR0002 |  Code    |  Error   | AwaitOrCaptureTasks
 
 ## Release 0.3
 
 ### Removed Rules
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|--------------------
-PCR0001 |  Code    |  Warning | Await used without specifying ConfigureAwait
+PCR0001 |  Code    |  Warning | Replaced by CA2007
+
+## Release 1.0
```

Comment:
Off topic: I like this way of doing release notes, it's very aligned with @ramonsmits ideas raised in https://github.com/Particular/CustomerSuccess/issues/1260 , is this something we should consider adopting across all repos?